README.md

```diff
@@ -50,7 +50,7 @@ The following things can't be fixed because of Bedrock limitations. They might b
 ## Compiling
 1. Clone the repo to your computer
 2. [Install Maven](https://maven.apache.org/install.html)
-3. Navigate to the Geyser root directory and run `git submodule update --init --recursive`. This downloads all the needed submodules for Geyser and is a crucial step in this process.
+3. Navigate to the Geyser root directory and run `git submodule update --init --recursive`. This command downloads all the needed submodules for Geyser and is a crucial step in this process.
 4. Run `mvn clean install` and locate to the `target` folder.
 
```

Comment:
Maybe you should add that if your having issues compiling try adding -U into the build command
E.g mvn clean install -U

Also you could add that adding -T 2C where 2 is the number of threads or cores (idk which one) maven uses

Reply:
@Camotoy thoughts? Barely out of the scope of this PR

Reply:
I would agree that is out of the scope of this PR. It's not a bad idea, however.